src/main/java/org/bankofspring/helloworld/annotation/HelloAnnotation.java

```diff
@@ -0,0 +1,30 @@
+package org.bankofspring.helloworld.annotation;
+
+import org.bankofspring.helloworld.HelloWorld;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+/**
+ * org.bankofspring.helloworld.HelloWorldProperty with annotation as 
+ * a (very) gentle introduction to wiring via annotation
+ * 
+ * @author Thierry Roisin
+ */
+// marks this POJO as a spring component , candidate to be "discovered"
+@Component
+public class HelloAnnotation implements HelloWorld {
```

Comment:
Pedantic - the others follow HelloWorld<type>. Could this be renamed to HelloWorldAnnotation?